fbw-common/src/wasm/extra-backend/lib/logging.h

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2022 FlyByWire Simulations
```

Comment:
Somehow I missed this comment and I'm surprised to see its from March.
But I like the idea and as described in my PR a better logging framework is planned for future iterations.
The one you suggested looks interesting - needs to be tested if it works in the sim. The ones I tried didn't due to the limitations of MSFS, for example because of "no exceptions" or  "no threads".